anonip.py

```diff
@@ -351,15 +398,20 @@ def parse_arguments(args):
         type=lambda x: _validate_integer_ht_0(x),
         help="assume IP address is in column n (1-based indexed; default: 1)",
     )
-    parser.set_defaults(column=[1])
     parser.add_argument(
         "-l",
         "--delimiter",
         metavar="STRING",
         type=str,
         help='log delimiter (default: " ")',
     )
-    parser.set_defaults(delimiter=" ")
+    parser.add_argument(
+        "--regex",
+        metavar="STRING",
+        nargs="+",
+        help="regex for detecting IP addresses (use optionally instead of -c)",
```

Comment:
does this help explicitly state that one can provide multiple whitespace separated regex?

Reply:
Yes. This will result in following help:

```
...
  --regex STRING [STRING ...]
                        regex for detecting IP addresses (use optionally instead of -c)
...
```